src/gui_common/art_gallery/GalleryViewer.tscn

```diff
@@ -31,12 +32,12 @@ GalleryCardModelScene = ExtResource( 4 )
 GalleryCardAudioScene = ExtResource( 6 )
 GalleryDetailsToolTipScene = ExtResource( 8 )
 
-[node name="MarginContainer" type="MarginContainer" parent="."]
+[node name="MarginContainer" type="MarginContainer" parent="." index="0"]
```

Comment:
What's up with these bunch of `index=` that got added in the scenes in this PR? If you reopen Godot and save all the scenes again, do these go away?

Reply:
No, it does not.

Reply:
I guess fine, we'll just have more and more of these in the project... I just checked and only a couple of our scene files currently suffer from this problem, this PR would at least double the number of files that have this strange thing going on.